cli/src/invocation/cmd/config.rs

```diff
@@ -0,0 +1,62 @@
+use crate::argparse::ConfigOperation;
+use crate::settings::Settings;
+use termcolor::{ColorSpec, WriteColor};
+
+pub(crate) fn execute<W: WriteColor>(
+    w: &mut W,
+    config_operation: ConfigOperation,
+    settings: &Settings,
+) -> anyhow::Result<()> {
+    match config_operation {
+        ConfigOperation::Set(key, value) => {
+            let filename = settings.set(&key, &value)?;
+            write!(w, "Set configuration value ")?;
+            w.set_color(ColorSpec::new().set_bold(true))?;
+            write!(w, "{}", &key)?;
+            w.set_color(ColorSpec::new().set_bold(false))?;
+            write!(w, " in ")?;
+            w.set_color(ColorSpec::new().set_bold(true))?;
+            write!(w, "{:?}.\n", filename)?;
```

Comment:
Hmm, not sure I agree with the clippy lint here (maybe a separate `write!(w, "\n")` would be clearer and appease the lint?)

However I wonder if it's worth investigating a neater way to do the coloured text formatting as `write!(w, "Set configuration value {} in {:?}", ...)` is drastically easier to read than the version with the coloured output

Reply:
Split this into #229 as it's unrelated to this change really

Reply:
I'm OK with appeasing clippy, though :)